index.html

```diff
@@ -1,278 +1,347 @@
+
+
 <!DOCTYPE html>
 
 <html lang="en">
 
 <head>
+
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>ibrahim's portfolio</title>
+    <title>ibrahim's Portfolio</title>
     <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
     <header>
```

Comment:
- [ ] While doing the **desktop version**, you mismatch the **mobile version** of the header section. Kindly fix that.
- Your website
![fdbfbdf](https://user-images.githubusercontent.com/26767383/155520030-2ec7dbd6-e495-49e3-a7fe-c85094b66f44.PNG)
- Figma Design 
![nfgnfgn](https://user-images.githubusercontent.com/26767383/155520068-9d640feb-0c6d-4170-aa89-d9d7782813d0.PNG)